routes/linkUrls.js

```diff
@@ -0,0 +1,33 @@
+const express = require('express');
+const router = express.Router();
+const cache = require('memory-cache');
+const asyncHandler = require('express-async-handler');
+
+const handleCache = require('../helpers/handleCache');
+const commonContent = require('../helpers/commonContent');
+
+router.get('/:codenames', asyncHandler(async (req, res, next) => {
+  await handleCache.evaluateCommon(res, ['urlMap']);
+
+  const KCDetails = commonContent.getKCDetails(res);
```

Comment:
```suggestion
  const kcDetails = commonContent.getKCDetails(res);
```
Unless it is at multiple places with wrong casing…

Reply:
Unfortunately, the wrong casing is in many places: 
![image](https://user-images.githubusercontent.com/11269013/63409420-6de88380-c3f1-11e9-86ec-05f3586d86dc.png)